src/main/java/edu/kis/vh/nursery/IStackInt.java

```diff
@@ -0,0 +1,14 @@
+package edu.kis.vh.nursery;
+
+public interface IStackInt {
```

Comment:
It does not look like an interface of a stack to me (push, pop, peek or top). 
It is a rhymer interface.